content/operate/rc/security/cidr-whitelist.md

```diff
@@ -13,15 +13,13 @@ weight: 30
 
 The [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) [allow list](https://en.wikipedia.org/wiki/Whitelist) lets you restrict traffic to your Redis Cloud database. When you configure an allow list, only the [IP addresses](https://en.wikipedia.org/wiki/IP_address) defined in the list can connect to the database. Traffic from all other IP addresses is blocked.
 
-## Database allow list
-
 You can configure your database's CIDR allow list to restrict client connections to a specific range of IP addresses.
```

Comment:
This reads like it just repeats what was said in the paragraph above. If the idea is to emphasise that you can specify a range (ie, with a start and end point) as well as individual IP addresses then you could probably just add that to the previous paragraph.

Reply:
```suggestion
```

Makes sense. Thanks!